src/main/kotlin/org/rust/ide/refactoring/functionSignature.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.refactoring
+
+import org.rust.lang.core.psi.RsFunction
+import org.rust.lang.core.psi.RsTypeParameter
+import org.rust.lang.core.types.type
+
+/**
+ * Helper class for storing and formatting information about the signature of a function.
+ */
+abstract class RsFunctionSignatureConfig(val function: RsFunction) {
+    protected val typeParametersText: String
+        get() {
+            val typeParams = typeParameters()
+            if (typeParams.isEmpty()) return ""
+            return typeParams.joinToString(separator = ",", prefix = "<", postfix = ">") { it.text }
```

Comment:
```suggestion
            return typeParams.joinToString(separator = ", ", prefix = "<", postfix = ">") { it.text }
```

Reply:
Should I also include the space after comma in `whereClausesText`?

Reply:
I think yes, so it would be consistent with default style of IntelliJ formatter and rustfmt